index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">    
+    <title>Portfolio</title>
+</head>
+<body>
+    <nav class="navbar">
+        <div class="logo">
+            <a href="">Alexander</a>
+        </div>
+        <ul class="top-navigation">
+            <li class="navigation-item">
+                <a href="">Portfolio</a>
+            </li>
+            <li class="navigation-item">
+                <a href="">About</a>
+            </li>
+            <li class="navigation-item">
+                <a href="">Contact</a>
+            </li>
+        </ul>
+        <div class="menu">
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+        </div>
+    </nav>
+
+    <main class="main" >
+        <div class="headline">
+            <div class="social-media">
+                <a href=""><img src="./ressources/icons/twitter.svg" alt="logo_twitter" 
+                    class="logo-social">
+                </a>
+                <a href=""><img src="./ressources/icons/github.svg" alt="logo_github" 
+                    class="logo-social">
+                </a>
+                <a href=""><img src="./ressources/icons/Linkedin.svg" alt="logo_Linkedin" 
+                    class="logo-social">
+                </a>
+                <a href=""><img src="./ressources/icons/angel.svg" alt="logo_angel" 
+                    class="logo-social">
```

Comment:
- [ ] Kindly wrap the headline section in a `<section>` tag.

Reply:
done 👍